ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,46 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# Information used to create certificates when creating the registry.
+cert_country: MyCountry
+cert_state: MyState
+cert_locality: MyCity
+cert_organization: MyCompany
+cert_origanizational_unit: MyBusinessUnit
```

Comment:
Do we want to set defaults for this? Should we not instead check if they have properly filled this out?

Reply:
i set these in case people forget. this is so it does not halt the playbook. We can set these to nothing and check that each has a value during execution.
What would you like to do?

Reply:
The way I would handle this is leaving these values commented out and then within your check yml file verify if those variables are defined. In the end, I don't think anyone wants a certificate with fake info anyway.

Reply:
ok, i will make the change as such.

Reply:
added and pushed